app/edge/components/groups-datatable/groupsDatatable.html

```diff
@@ -0,0 +1,131 @@
+<div class="datatable">
+  <rd-widget>
+    <rd-widget-header icon="fa-plane-departure" title-text="Edge Groups"> </rd-widget-header>
```

Comment:
Icon should be injected via binding.

Reply:
why?

Reply:
fixed, but would love to know the reasoning